c7n/resources/account.py

```diff
@@ -1428,14 +1413,16 @@ def validate(self):
                        'AwsKmsKeyId']:
                 attrs.add(key)
         self.multi_attrs = attrs
-        return super(GlueEncryptionEnabled, self).validate()
+        return super(GlueCatalogEncryptionEnabled, self).validate()
 
     def get_target(self, resource):
         if self.annotation in resource:
             return resource[self.annotation]
         client = local_session(self.manager.session_factory).client('glue')
-        encryption_setting = client.get_data_catalog_encryption_settings().get(
-            'DataCatalogEncryptionSettings')
+        encryption_setting = resource.get('DataCatalogEncryptionSettings')
+        if str(self.manager.get_model()) != '<TypeInfo AWS::Glue::Catalog>':
```

Comment:
Thinking more about it, I'm not sure if this is the right way to do it

Reply:
if you get to the self.manager.type is the simpler comparison

Reply:
Thanks! That looks a lot cleaner